docs/blueprints/storage-node-downtime-disqualification.md

```diff
@@ -0,0 +1,112 @@
+
+    - Refactor relevant overlay methods to handle downtime_suspended nodes: `KnownReliable`, `FindNodes`, etc.
+    - Add notification to SN dashboard indicating suspension for downtime.
+
+4. Implement email notifications if node becomes suspended
+
+5. Implement evaluation chore to reinstate and disqualify nodes
+    - Check total downtime for each suspended node where last_contact_success > last_contact_failure. If the total downtime has fallen below the allowed downtime, the node's suspension is lifted, though it still remains under review.
+    - If node.under_review <= now - (grace period + tracking period) it is eligible for evaluation
+    - If the node is eligible for evaluation, we must also ensure that the entire tracking period is accounted for. If last contact was a failure and occurred before the end of the tracking period, we might have a window of downtime which has not yet been recorded. Thus, to measure all downtime for the tracking period, we must also wait until last_contact_success > failure OR last_contact_failure >= end of tracking period.
+
+        When these conditions are met, measurements should take care to include all downtime within the period. In addition to the lower bound issue explained above under step 3, we also have an upper bound issue in this case. We may have an entry which exists beyond the upper bound of the tracking period, yet holds some amount of downtime which occured within it.<br>
+        <br>
+        For example, if we want to measure the total downtime for node A from March 1st 00:00 to March 31st 00:00, and we have an entry from April 1st 01:00 indicating 2 hours of downtime, we must include the one hour of downtime which occurred within the tracking period in our calculation. 
+
+## Wrapup
+
+- The person that implements step 5 above should archive this document.
+
+- The [Disqualification blueprint](disqualification.md), and possibly the whitepaper, will need to be updated to reflect new up/downtime disqualification mechanic.
+
+- Edit [Audit Suspension blueprint](audit-suspend.md) to reflect change from `suspended` to `audit_suspended`
+
+- Link to this document in [Downtime Tracking](storage-node-downtime-tracking.md)
+
+## Open issues
+
+- There is a way to get around downtime tracking in its current implementation. Only come online long enough to check in then immediately shut down. Repeat this process frequently enough and a node can effectively remain offline and never be punished for it. This is due to the way downtime tracking selects nodes where last contact success < last contact failure. By checking in frequently enough, a node can reduce the risk that its last contact was a failure when the estimation chore runs.
+
+- It is possible for a node to continuously cycle through suspension and reinstatement. How frequently this could happen depends upon the length of the tracking and grace periods. Should there be a maximum number suspensions before disqualification?
```

Comment:
I think we need this. Otherwise, nodes can drive up repair load by cycling through suspension and reinstatement, right?

Reply:
Yeah, this also follows the "going in and out of suspension, but still under review" question. Which, some in the forum and here have suggested that the pros may outweigh the cons https://github.com/storj/storj/pull/3834#discussion_r400557174
Though I think it's still a valid concern